source/index.html

```diff
@@ -766,6 +775,99 @@ <h3 class='title'>Pricing</h3>
 				<button id='question-pricing-button'>Details</button>
 			</div>
 		</section>
+		
+		 <section id='addonsbanner'>
+			<style>
+				#addonsbanner {
+					grid-column: 1;
+					grid-row: 13;
+					display: flex;
+					justify-content: space-around;
+					align-items: stretch;
+					row-gap: 50px;
+					padding-top: 20px;
+					padding-bottom: 20px;
+				}
+
+				
+				
+				@media screen and (max-width: 680px) {
+					#addonsbanner {
+						flex-direction: column;
+						align-items: center;
+					}
+					#addonsbanner > .addonsbanner {
+						grid-column: 1;
+						grid-row: 1;
+
```

Comment:
```suggestion
```
Empty lines should be intentional and uniformly applied throughout the file.  The rest of this file doesn't have a trailing newline inside style blocks, so we shouldn't here either.